src/uopool/services/uopool.rs

```diff
@@ -81,7 +85,13 @@ where
         Ok(())
     }
 }
-
+fn get_addr(bytes: &Bytes) -> Option<Address> {
```

Comment:
This function can also be used in some other places (sanity checks, simulation )

Reply:
Can be moved to utils